### ISSUE TITLE
Dashboard not showing in Safari

### ISSUE THREAD

**Issue by Mariusthvdb** [hacs/integration#3914]
### Web browser

Safari Mac 

### Web browser version

17.5 (19618.2.12.11.6)

## System Information

version | core-2024.7.3
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.12.4
os_name | Linux
os_version | 6.6.29-haos
arch | x86_64
timezone | Europe/Amsterdam
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
HACS Data | ok
GitHub API Calls Remaining | 4946
Installed Version | 2.0.0-dev-43e5c89
Stage | running
Available Repositories | 1398
Downloaded Repositories | 44

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | true
-- | --
subscription_expiration | 31 augustus 2024 om 02:00
relayer_connected | true
relayer_region | eu-central-1
remote_enabled | true
remote_connected | true
alexa_enabled | false
google_enabled | true
remote_server | eu-central-1-5.ui.nabu.casa
certificate_status | ready
instance_id | f51092d0098f40faa50dfd391c4bf925
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 12.3
-- | --
update_channel | beta
supervisor_version | supervisor-2024.06.2
agent_version | 1.6.0
docker_version | 25.0.5
disk_total | 458.4 GB
disk_used | 27.7 GB
healthy | true
supported | true
host_connectivity | true
supervisor_connectivity | true
ntp_synchronized | true
virtualization | 
board | generic-x86-64
supervisor_api | ok
version_api | ok
installed_addons | Log Viewer (0.17.0), Samba share (12.3.1), Terminal & SSH (9.14.0), Mosquitto broker (6.4.1), Z-Wave JS (0.6.2), ESPHome (2024.7.0), Studio Code Server (5.15.0), Home Assistant Google Drive Backup (0.112.1), Get HACS (1.2.0)

</details>

<details><summary>Dashboards</summary>

dashboards | 10
-- | --
resources | 23
views | 117
mode | yaml

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | 8 juli 2024 om 07:16
-- | --
current_recorder_run | 22 juli 2024 om 17:22
estimated_db_size | 2839.53 MiB
database_engine | sqlite
database_version | 3.45.3

</details>

### Checklist

- [X] I'm running the newest version of HACS <https://github.com/hacs/integration/releases/latest>
- [X] I have filled out the issue template to the best of my ability.
- [X] I have read <https://hacs.xyz/docs/issues>
- [X] This issue is related to the frontend of HACS.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This is a bug and not a feature request.
- [X] This issue is not a duplicate issue of currently [open](https://github.com/hacs/integration/issues) or issues [pending release](https://github.com/hacs/integration/issues?q=is%3Aissue+is%3Aclosed+sort%3Aupdated-desc+milestone%3Anext).

### Describe the issue

as title, /hacs/dashboard is not showing on Mac Safari, running fine on Mac Chrome

### Reproduction steps

1. Install in Safari
2. navigate to /hacs/dashboard
3. see empty view
...

### Screenshots

<img width="810" alt="Scherm­afbeelding 2024-07-22 om 17 40 15" src="https://github.com/user-attachments/assets/ca9dad70-911a-4ecc-b527-7d44863b0d8f">

<img width="729" alt="Scherm­afbeelding 2024-07-22 om 18 06 14" src="https://github.com/user-attachments/assets/8889be3f-2d3f-4aa4-8a01-d4a60b81c5ae">

### Javascript logs from your browser console

```text
Unhandled Promise Rejection: TypeError: undefined is not an object (evaluating 'this.columns[this._sortColumn].title')
```

### Diagnostics dump

[config_entry-hacs-b74d20b6c11624dfe614b8dd624cf670.json](https://github.com/user-attachments/files/16336636/config_entry-hacs-b74d20b6c11624dfe614b8dd624cf670.json)

**Comment by ludeeus** [comment]
Unable to reproduce, try clearing browser cache.

**Comment by Mariusthvdb** [comment]
I had ofc done that many times, even restarted the browser. but the view remained empty.

but, just now I deleted the 4 HACS entries in local storage manually, and after another reload, the HACS view is now showing

2 entires returned 

<img width="793" alt="Scherm­afbeelding 2024-07-22 om 22 16 44" src="https://github.com/user-attachments/assets/084ec097-91e0-4a70-89f8-ce092cb9d67f">

to show what I deleted .

I still see the Grouping return to Downloaded on each and every revisit, no matter how often I select Dont group.

<img width="844" alt="Scherm­afbeelding 2024-07-22 om 22 18 16" src="https://github.com/user-attachments/assets/000ded1a-40f6-46f0-82dc-82c78fe2901f">

That is browser agnostic though, and on all of my test apps (Chrome, Safari, Firefox, Edge).
probably another matter, if not simply a HA issue...

closing
